tox.ini

```diff
@@ -0,0 +1,31 @@
+[tox]
+minversion = 1.6
+skipdist = True
+envlist = py27,py36,py37,pep8,cover,docs
```

Comment:
Should probably also include py34 and py35 so long as those continue to be supported by ncclient